autoscale/src/main/java/com/sequenceiq/periscope/controller/AutoScaleClusterCommonService.java

```diff
@@ -219,6 +225,15 @@ protected Cluster syncCBClusterByName(String stackName) {
                 .orElseThrow(NotFoundException.notFound("cluster", stackName));
     }
 
+    public boolean enableStopStartBasedOnEntitlement(Cluster cluster, DistroXAutoscaleClusterRequest autoscaleClusterRequest) {
```

Comment:
This method isn't actually enabling anything, so the method name is misleading.

shouldEnableStopStartBasedOnEntitlement is a better name.

Reply:
Okay will change it.

Reply:
Changed